cou/cli.py

```diff
@@ -99,4 +100,5 @@ def entrypoint() -> int:
         return 0
     except Exception as exc:  # pylint: disable=broad-exception-caught
         logging.error(exc)
+        logging.error(traceback.format_exc())
```

Comment:
```suggestion
        logging.exception(error)
```

Reply:
I didn't know about `logging.exception`. Thanks it's very helpful and cleaner.